roles/maas_firewall/tasks/teardown.yaml

```diff
@@ -0,0 +1,16 @@
+---
+# Remove iptables rules as part of teardown
+
+- name: Set policy for INPUT chain to accept
+  ansible.builtin.iptables:
+    chain: INPUT
+    policy: ACCEPT
+
+- name: Set policy for FORWARD chain to drop
+  ansible.builtin.iptables:
+    chain: FORWARD
+    policy: ACCEPT
+
+- name: Flush all other rules
```

Comment:
Flushing the rules first locks you out of the host instantly since the policies for forward and input are set to drop (at least, it did when I tried.. ).

Output never gets set to drop, so it seemed to be OK not add it here?

Reply:
hmm interesting, I'm curious how the above rules don't get flushed then

Reply:
I'm not sure either- though from what I gathered from the manual + online resources flush doesn't reset the policies to accept but only clears out user-added rules. I can't find very clear documentation on this though?